src/app/products/products-routing.module.ts

```diff
@@ -229,11 +230,21 @@ const routes: Routes = [
             },
             {
               path: 'tax-groups',
-              component: ManageTaxGroupsComponent,
-              resolve: {
-                taxGroups: ManageTaxGroupsResolver
-              },
               data: { title: extract('Manage Tax Groups'), breadcrumb: 'Tax Groups'},
+              children: [
+                {
+                  path: '',
+                  component: ManageTaxGroupsComponent,
+                  resolve: {
+                      taxGroups: ManageTaxGroupsResolver
+                  }
+                },
+                {
+                  path: 'create',
+                  component: CreateTaxGroupComponent,
+                  data: {title: extract('Create Tax Group'), breadcrumb: 'Create'},
```

Comment:
Ensure proper indentation here and make correction to the breadcrumb.
```suggestion
                  data: { title: extract('Create Tax Group'), breadcrumb: 'Create Tax Group' },
```